fix(mgmt): avoid 500 error when hocon syntax error (#12940)
emqx/emqx: changes/ce/feat-12940.en.md

Add `ignore_readonly` argument to `PUT /configs` API.

Prior to this change, EMQX would retrun 400 (BAD_REQUEST) if the raw config
included readonly root keys (`cluster`, `rpc`, and `node`).

After this enhancement it can be called as `PUT /configs?ignore_readonly=true`,
EMQX will in this case ignore readonly root config keys, and apply the rest.

> A warning message is logged to warn that the readonly keys are dropped.

Reviewer: ```suggestion
For observability purposes, an informational message is logged if any readonly keys are dropped.
```
?

Author: thanks, fixed.
